modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js

```diff
@@ -12,6 +12,8 @@ const PaymentMethod = () => {
 	const [agree, setAgree] = useState(false);
 	const [{orderId, product}] = useContext(SelectedQuoteContext);
 	const [methods, setMethods] = useState([]);
+	const productDiscounted = product.price * 0.95;
```

Comment:
Hi @samuelleolopes 

I'm not sure, but I think this product.price is coming as a string.

And you are multiplying then. Something like '390' * 0.95

Javascript still evaluate the value, but it not trustful

```suggestion
	const productDiscounted = Number(product.price) * 0.95;
```

Reply:
Done